CodeEditModules/Modules/AppPreferences/src/Model/Source Control/SourceControlPreferences.swift

```diff
@@ -40,9 +40,9 @@ public extension AppPreferences {
         /// Indicates whether or not we should include the upsteam
         public var includeUpstreamChanges: Bool = false
         /// The selected value of the comparison view
-        public var comparisonView: Int = 0
+        public var comparisonView: ComparisonView = .localLeft
```

Comment:
Since this is a setting which will be in `preferences.json` make sure to choose a name that is descriptive to what it does.

Reply:
What name do you like best?

Reply:
Maybe something like `revisionComparisonLayout`

Reply:
Do you mean such a thing?
`public var revisionComparisonLayout: ComparisonView = .localLeft`

Reply:
yes and make sure to change the enum name accordingly